.travis.yml

```diff
@@ -1,27 +1,35 @@
-language: node_js
+matrix:
+  include:
+    -
+      language: node_js
+      node_js:
+        - node
+      addons:
+        apt:
+          packages:
+            - python-virtualenv
+      after_success:
+        - "npm run coveralls"
+    -
+      language: python
+      python:
+        - "3.6"
```

Comment:
You have to request a specific distribution to get 3.7: https://docs.travis-ci.com/user/languages/python/#python-37-and-higher. If you're cool with specifying the distribution (I am), we can do 3.7. I would like to keep 3.6 as an additional job though since Python versioning has already bitten us recently with `fathom-serve`.

Reply:
I'm loath to delay landing this important PR any longer than absolutely necessary, so let's leave it as is and add 3.7 later, as a separate PR, if it gives us anything. If we have to be 3.6-compatible anyway (which I agree we seem to have to, practically), I don't see any benefit to testing on 3.7 at all—breaking changes are rare in CPython.

Reply:
That is a good point about the lack of benefit from testing on 3.7 if we're testing on 3.6. I'll remove the 3.7 job.

Reply:
And now I see you went ahead and added it, and it works. Great. I don't even care if it adds anything, because I'm so psyched to get this patch landed. :-)

Reply:
Haha, okay. It'll stay!